app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= food.name %></td>
+  <td><%= food.measurement_unit %></td>
+  <td>$<%= food.price %></td>
+  <td><%= button_to 'Destroy', food, method: :delete, data: { confirm: 'Are you sure?' } %></td>
```

Comment:
- [ ] Just like I mentioned in the video, I think it's quite important to use `Delete` as contained in the wireframe so that you match requirements fully, also this will enhance the user experience by improving user's understanding and navigation.
- [ ] [OPTIONAL] If you check the wireframe, the color of this button is different :wink: How about trying another color on this too, to distinguish from other texts in black :+1: